nixos/modules/system/boot/loader/systemd-boot/systemd-boot-builder.py

```diff
@@ -190,12 +263,19 @@ def main():
     mkdir_p("@efiSysMountPoint@/efi/nixos")
     mkdir_p("@efiSysMountPoint@/loader/entries")
 
+    if @signed@ == 1:
```

Comment:
I've never seen this python syntax, what is this?

Reply:
This is probably the ugliest part of the whole PR. I'll improve it to make it more clear.

Reply:
Quoted.